docs/local_install_docker_compose.md

```diff
@@ -1,102 +1,139 @@
-# Local install of Substra (macOS; UNIX like system)
-
-This guide will help you to get Substra source code and to start a local instance with 2 organisations: owkin and chu-nantes.
-
-## Dependencies
-
-- docker: [Docker Desktop](https://www.docker.com/products/docker-desktop)
-- python 3 (recommended 3.6 or 3.7)
-  - it is recommended to use a virtual environment to install Substra packages (for instance [virtualenv](https://virtualenv.pypa.io/en/latest/))
-- yarn
-- redis server
-
-## Configuration
-
-- docker:
-  - on macOS, by default, docker has access only to the user directory.
+# Local install of Substra (macOS & Ubuntu/Debian)
+
+- [Local install of Substra (macOS & Ubuntu/Debian)](#local-install-of-substra-macos--ubuntudebian)
```

Comment:
I find that it would be nice to try to follow as much as possible the sections in the skaffold doc.
In particular would be nice to have a section about credentials/url as they are not the same as for the skaffold setup